plugins/scaffolder-common/api-report.md

```diff
@@ -64,5 +65,6 @@ export const templateEntityV1beta3Validator: KindValidator;
 export type TemplateInfo = {
   entityRef: string;
   baseUrl?: string;
+  entityMetadata: EntityMeta;
```

Comment:
I think that this should be optional. There's a chance that new code could pick up old tasks which don't have it set. So let's make it optional for now to enforce runtime checking of the existence for now.

Maybe also let's have it on something like this shape?

```ts
export type TemplateInfo = {
  entityRef: string;
  entity?: {
    metadata?: EntityMeta;
  }
}
```

Reply:
This way it's similar to the `${{ user.entity.metadata }}` helper too.

Reply:
@benjdlambert thank you for the feedback, and I have implemented the changes.